main.js

```diff
@@ -4,16 +4,30 @@ import Add from './modules/add.js';
 
 const add = new Add();
 const remove = new Remove();
-const createNew = document.getElementById('btn');
-const bookContainer = document.getElementById('book-list');
+// eslint-disable-next-line
```

Comment:
- [ ] Please do not disable linters unless it is mentioned that it is not a project requirement. Linters improved your code quality and are also nice automatic debugging tools. :pray: